editing/edit-with-branch-versioning/README.md

```diff
@@ -0,0 +1,68 @@
+# Edit with branch versioning
+
+Create, query and edit a specific server version using service geodatabase.
+
+![Image of edit with branch versioning](EditWithBranchVersioning.png)
+
+## Use case
+
+Workflows often progress in discrete stages, with each stage requiring the allocation of a different set of resources and business rules. Typically, each stage in the overall process represents a single unit of work, such as a work order or job. To manage these, you can create a separate, isolated version and modify it. Once this work is complete, you can integrate the changes into the default version.
+
+## How to use the sample
+
+Upon opening the sample, you will be prompted to enter credentials for the service (un: editor01 / pwd: editor01.password). Once loaded, the map will zoom to the extent of the feature layer. The current version is indicated in the top left corner of the map. You can create a new version by specifying the version information (name, access, and description) in the form in the top right corner, and then clicking "Create version". See the *Additional Information* section for restrictions on the version name.
+
+Select a feature to edit an attribute and/or click again with the right mouse button to relocate the point.
+
+Click the "Switch version" button in the top left corner to switch back and forth between the version you created and the default version. Edits will automatically be applied to your version when switching to the default version.
+
+## How it works
+
+1. Create and load a `ServiceGeodatabase` with a feature service URL that has enabled Version Management.
+2. Get the `ServiceFeatureTable` from the service geodatabase.
+3. Create a `FeatureLayer` from the service feature table.
+4. Create `ServiceVersionParameters` with a unique name, `VersionAccess`, and description.
+    * Note - See the additional information section for more restrictions on the version name.
+5. Create a new version calling `ServiceGeodatabase.createVersionAsync()` passing in the service version parameters.
+6. Connect to a `ListenableFuture<ServiceVersionInfo>` to obtain the ServiceVersionInfo of the version created.
```

Comment:
I would probably phrase this a little differently.
Perhaps 
```
Retreive the result of the async call, to obtain the `ServiceVersionInfo` of the version created.
```

Also `ServiceVersionInfo` wasn't mentioned yet, so I think it can be in backticks